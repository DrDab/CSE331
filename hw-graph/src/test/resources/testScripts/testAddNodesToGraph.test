# Create graphs w/ 0, 1, 2 nodes.
CreateGraph g0

CreateGraph g1
AddNode g1 ALPHA

CreateGraph g2
AddNode g2 ALPHA
AddNode g2 BRAVO

# Try adding 1, 2, 3 edges to graph w/ 0 nodes
AddNode g0 ALPHA
ListNodes g0
AddNode g0 BRAVO
ListNodes g0
AddNode g0 CHARLIE
ListNodes g0

# Try adding 1, 2, 3 edges to graph w/ 1 node
AddNode g1 BRAVO
ListNodes g1
AddNode g1 CHARLIE
ListNodes g1
AddNode g1 DELTA
ListNodes g1

# Try adding 1, 2, 3 edges to graph w/ 2 nodes
AddNode g2 CHARLIE
ListNodes g2
AddNode g2 DELTA
ListNodes g2
AddNode g2 ECHO
ListNodes g2
