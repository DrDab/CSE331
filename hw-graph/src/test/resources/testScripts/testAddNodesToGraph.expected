# Create graphs w/ 0, 1, 2 nodes.
created graph g0

created graph g1
added node ALPHA to g1

created graph g2
added node ALPHA to g2
added node BRAVO to g2

# Try adding 1, 2, 3 nodes to graph w/ 0 nodes
added node ALPHA to g0
g0 contains: ALPHA
added node BRAVO to g0
g0 contains: ALPHA BRAVO
added node CHARLIE to g0
g0 contains: ALPHA BRAVO CHARLIE

# Try adding 1, 2, 3 nodes to graph w/ 1 node
added node BRAVO to g1
g1 contains: ALPHA BRAVO
added node CHARLIE to g1
g1 contains: ALPHA BRAVO CHARLIE
added node DELTA to g1
g1 contains: ALPHA BRAVO CHARLIE DELTA

# Try adding 1, 2, 3 nodes to graph w/ 2 nodes
added node CHARLIE to g2
g2 contains: ALPHA BRAVO CHARLIE
added node DELTA to g2
g2 contains: ALPHA BRAVO CHARLIE DELTA
AddNode g2 ECHO
g2 contains: ALPHA BRAVO CHARLIE DELTA ECHO
